rune/README.md

```diff
@@ -116,5 +122,10 @@ cd "$HOME/rune_workdir/rune-container"
 sudo rune run ${Occlum_application_container_name}
 ```
 
+## Run WAMR application
+
+Please refer to [this guide](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/product-mini/platforms/linux-sgx/enclave-sample/App/wamr-bundle.md) to execute WAMR container in OCI bundle way.
+
```

Comment:
Please remove `## Run WAMR application` section.   `Developement` section is taking Occlum as example. There is no need to add  how to execute WAMR container in OCI bundle way.